### ISSUE TITLE
Function names are cut off after the first line

### ISSUE THREAD

**Issue by JWWeatherman** [JWWeatherman/math_bot#100]
On a mac using Chrome I created a function named "Dance Around" but as you can see in the image only the first word is visible.

![cursor_and_math_bot](https://user-images.githubusercontent.com/32912678/41492325-76f735ee-70cc-11e8-901b-d97804a740bf.jpg)

**Comment by smh69** [comment]
tl;dr: not fun to change css AND html, but at first glance, it seems necessary.

I think it's just wrapping down below, but the overflow is hidden, notice the whitespace for your Dance and Go to appear to extend lower than others, without their rounded corners on the bottoms?  This seems like a great place for a css tooltip, where you hover and can see the full name of the function.  Other places like planets on left side of previous page, or the steps of each planet on right side of previous page could also use a similar trick.  However, how to implement the simplest change is unknown to me at first glance.  In the planets sections, you have a separate class (planet-1, planet-2, planet-n) for each planet so might select off that for a long description.  As for the steps, they'd be different, because they dont have unique class to use as a selector (and maybe less useful because YOU control the steps so you can avoid long names... still a long description might be nice).  As for the function names, since they're controlled by user, it's ideal (I noticed wanting this, instead of having to click to see function contents and long name, I ended up using abbreviations instead) to use a hover-tooltip, but again, no way to key off of existing css classes so might require altering html or computed html if done in js.

**Comment by smh69** [comment]
as of 20180704 can confirm that hovering over function contents (where each operation/func is represented with an icon but no label underneath) shows a hint of what function it is -- in production. very helpful.
